packages/valory/skills/decision_maker_abci/models.py

```diff
@@ -246,13 +246,6 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.use_fallback_strategy: bool = self._ensure(
             "use_fallback_strategy", kwargs, bool
         )
-        if self.trading_strategy not in [
-            STRATEGY_BET_AMOUNT_PER_CONF_THRESHOLD,
```

Comment:
Are these constans used anywhere else? Otherwise, remove from the heading of the file. 
Also, why `STRATEGY_BET_AMOUNT_PER_CONF_THRESHOLD = "bet_amount_per_conf_threshold"`  and not ` bet_amount_per_threshold`? 
See https://github.com/valory-xyz/trader/blob/b012dd31f0c34bd2e2aaadf90dc078bf05fb533e/packages/valory/agents/trader/aea-config.yaml#L200

Reply:
943c34b383cd616d6b15a3793fac82a097c93381